.idea/.name

```diff
@@ -1 +1 @@
-genequery-kotlin
```

Comment:
I guess the reason why IDEA is trying to rename the project is your folder's structure.
![image](https://cloud.githubusercontent.com/assets/1048393/17135171/15fe83be-5337-11e6-9ef4-00c62657a069.png)
`genequery-kotlin` must be the root folder of the project (even if its name differs from the original), but it is not. Try to make it such.